crates/pink/capi/src/v1/mod.rs

```diff
@@ -58,6 +58,7 @@ pub mod ecall {
         pub gas_price: Balance,
         pub deposit_per_item: Balance,
         pub deposit_per_byte: Balance,
+        pub default_deposit_limit: Balance,
```

Comment:
The v1 API is frozen. So we can not change this.
Let's just make this into a const ConfigValue.